netbox/dcim/forms/connections.py

```diff
@@ -82,14 +83,22 @@ def __new__(mcs, name, bases, attrs):
 
     class _CableForm(CableForm, metaclass=FormMetaclass):
 
-        def __init__(self, *args, **kwargs):
+        def __init__(self, *args, initial=None, **kwargs):
+
+            initial = initial or {}
```

Comment:
Would it work to just do initial={} in the function def?

Reply:
It would, but we generally want to [avoid passing mutable objects as arguments](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments).